### PR TITLE
Adding code to run on Powersystem

### PR DESCRIPTION
**Pull request by genisysram**
Hi
Here is my contribution to your code, its working good on powersystems.

Thanks for the code, its working good.
## What do these changes do?
Added Architecture "ppc64le"
## Are there changes in behavior for the user?
No

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,7 @@ cache:
   npm: true
   ccache: true
 
-node_js:
-  - stable
```
Comment (BYVoid) [inline .travis.yml:8]: Why remove 'stable'?

Comment (genisysram) [reply .travis.yml:8]: Stable is failing both in amd64 and ppc64le
https://travis-ci.org/github/genisysram/OpenCC/builds/746974897

Comment (BYVoid) [reply .travis.yml:8]: This seems a problem we should fix, rather than removing 'stable'.

**Comment by BYVoid** [comment]
.travis.yml will be removed.

**Pull request closed without merging**